config-nfs.md

```diff
@@ -50,7 +50,7 @@ matching clients can access this export only in read-only mode
 matching clients can access this export only in read-write mode
 
 **noacl**  
-dCache ACLs will be ignored; only posix access permissions will be considered. This is the default.
+dCache ACL's will be ignored; only posix access permissions will be considered. This is the default.
```

Comment:
Actually, the Oxford Dictionaries appear to disagree. https://en.oxforddictionaries.com/punctuation/apostrophe , see the MPs example.

It appears that even in German it should be ACLs. https://de.wikipedia.org/wiki/Apostroph , see the CDs example.